src/components/Navigator/NavigatorCard.vue

```diff
@@ -835,7 +859,16 @@ export default {
     setActiveUID(uid) {
       this.activeUID = uid;
       this.resetScroll = false;
-      sessionStorage.set(STORAGE_KEYS.activeUID, uid);
+    },
+    /**
+     * Handles the `navigate` event from NavigatorCardItem, guarding from selecting an item,
+     * that points to another technology.
+     */
+    handleNavigationChange(uid) {
```

Comment:
If a user doesn't navigate through the technology (they go straight away to a new different technology instead by just filtering) we never set the activeUID and therefore we cannot persist the filtering data :(

We need a solution also for those cases.

Reply:
It was called, but I had some mistakes in my logic. Should be fixed now.